pkg/vm/stackitem/serialization_test.go

```diff
@@ -144,7 +144,7 @@ func TestSerialize(t *testing.T) {
 		for i := 0; i < MaxArraySize; i++ {
 			m.Add(Make(i), zeroByteArray)
 		}
-		testSerialize(t, nil, m)
+		// testSerialize(t, nil, m) // It contains too many elements already, so ErrTooBig.
```

Comment:
Decreasing cycle upper limit (probably to `(MaxDeserialized-1)/2` will allow to still test this.

Reply:
Fixed.